### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by secondsun**
I've done a material design refresh.  New steps will be to achieve feature parity with the Cordova lite app.

### CODE REVIEW

**Comment by secondsun** [comment]
@wei-lee @danielpassos :eyeglasses: ?

**Comment by secondsun** [comment]
Here is a demo : https://drive.google.com/a/redhat.com/file/d/0B64fta5NYF0TOW9pVXBKSm1LNkU/view?usp=sharing

**Comment by wei-lee** [comment]
:fireworks: nice!

File: app/build.gradle [hunk 1]
```diff
@@ -30,7 +30,9 @@ license {
 }
 
 dependencies {
-  compile fileTree(dir: 'libs', include: ['*.jar'])
-  compile 'com.android.support:appcompat-v7:22.2.1'
-  compile 'com.feedhenry:fh-android-sdk:2.4.0-SNAPSHOT'
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:appcompat-v7:23.0.1'
+    compile 'com.android.support:recyclerview-v7:23.0.+'
+    compile 'com.feedhenry:fh-android-sdk:3.0.0-SNAPSHOT'
```
Comment (danielpassos) [inline app/build.gradle]: Any reason for bump to `3.0.0-SNAPSHOT` instead of `2.4.0`?

File: app/src/main/assets/fhconfig.properties [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
-projectid = kdwlfamzejuqwxg5nykpwzpp
-appkey = 279f99742ac0697b945deb4b3428b23be5ce89bb
-connectiontag = 0.0.3
\ No newline at end of file
+host = https://testing.feedhenry.me
+appid = l5z254um56q52pbqnmv5sqzf
+projectid = l5z254vsghfpliwin3z5v4zo
+appkey = 9e94fbb05e71f201754a6d1de205cb3524b6b37
+connectiontag = 0.0.8
+
```
Comment (danielpassos) [inline app/src/main/assets/fhconfig.properties]: Let's use the old credentials so, people can use it online, or let's remove credentials.

File: app/src/main/java/redhat/com/syncsample/item/ShoppingItem.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ * Please refer to your contract with FeedHenry for the software license agreement.
+ * If you do not have a contract, you do not have a license to use this software.
+ */
+package redhat.com.syncsample.item;
+
+import java.io.Serializable;
+
+/**
+ * Created by summers on 10/13/15.
+ */
```
Comment (danielpassos) [inline app/src/main/java/redhat/com/syncsample/item/ShoppingItem.java]: Let's remove author

File: app/src/main/java/redhat/com/syncsample/item/ShoppingItemAdapter.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import redhat.com.syncsample.R;
+
+/**
+ * Created by summers on 10/13/15.
+ */
```
Comment (danielpassos) [inline app/src/main/java/redhat/com/syncsample/item/ShoppingItemAdapter.java]: Let's remove author

File: app/src/main/java/redhat/com/syncsample/view/CreateItemDialogFragment.java [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+import android.widget.EditText;
+
+import java.util.Date;
+
+import redhat.com.syncsample.R;
+import redhat.com.syncsample.item.ShoppingItem;
+
+/**
+ * Created by summers on 10/13/15.
+ */
```
Comment (danielpassos) [inline app/src/main/java/redhat/com/syncsample/view/CreateItemDialogFragment.java]: Let's remove author

File: app/src/main/res/anim/dialog_slide_down.xml [hunk 6]
```diff
@@ -7,9 +7,7 @@
...
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent"
-    android:padding="10dp"
-    android:textSize="20sp" >
-</TextView>
\ No newline at end of file
+<translate xmlns:android="http://schemas.android.com/apk/res/android"
+    android:duration="@android:integer/config_shortAnimTime"
+    android:fromYDelta="0%p"
+    android:toYDelta="100%p" />
```
Comment (danielpassos) [inline app/src/main/res/anim/dialog_slide_down.xml]: Let's add newline end of file

File: app/src/main/res/anim/dialog_slide_up.xml [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+    Copyright (c) 2015 FeedHenry Ltd, All Rights Reserved.
+
+    Please refer to your contract with FeedHenry for the software license agreement.
+    If you do not have a contract, you do not have a license to use this software.
+
+-->
+<translate xmlns:android="http://schemas.android.com/apk/res/android"
+    android:fromYDelta="50%p" android:toYDelta="0%p"
+    android:duration="@android:integer/config_shortAnimTime"/>
```
Comment (danielpassos) [inline app/src/main/res/anim/dialog_slide_up.xml]: Let's add newline end of file

File: app/src/main/res/layout/edit_details_dialog.xml [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Save"
+        style="@style/ButtonStyle"
+        android:id="@+id/save_button"
+        android:layout_alignParentBottom="true"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentEnd="true" />
+
+</RelativeLayout>
```
Comment (danielpassos) [inline app/src/main/res/layout/edit_details_dialog.xml]: Let's add newline end of file

File: app/src/main/res/layout/save_details_dialog.xml [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Save"
+        style="@style/ButtonStyle"
+        android:id="@+id/save_button"
+        android:layout_alignParentBottom="true"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentEnd="true" />
+
+</RelativeLayout>
```
Comment (danielpassos) [inline app/src/main/res/layout/save_details_dialog.xml]: Let's add newline end of file

File: app/src/main/res/layout/shopping_item.xml [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+        android:paddingBottom="@dimen/activity_vertical_margin"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1px"
+        android:background="#e0e0e0"
+        />
+</LinearLayout>
```
Comment (danielpassos) [inline app/src/main/res/layout/shopping_item.xml]: Let's add newline end of file

File: app/src/main/res/values/colors.xml [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+
+    Please refer to your contract with FeedHenry for the software license agreement.
+    If you do not have a contract, you do not have a license to use this software.
+
+-->
+<resources>
+    <color name="redhat_red">#CC0000</color>
+    <color name="redhat_dark_red">#781f1c</color>
+    <color name="redhat_accent">#564979</color>
+</resources>
```
Comment (danielpassos) [inline app/src/main/res/values/colors.xml]: Let's add newline end of file

File: app/build.gradle (hunk 1 above)
Comment (secondsun) [inline app/build.gradle]: We are going to be fixing a bucket of bugs in the SDK.  No reason to link against the broken one.

File: app/src/main/assets/fhconfig.properties (hunk 2 above)
Comment (secondsun) [inline app/src/main/assets/fhconfig.properties]: +1 Remove credentials

**Pull request merged**